core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionNetworkOutput.java

```diff
@@ -164,7 +164,7 @@ public boolean sendSync(Transmission transmission) {
                 // POST the transmission data to the endpoint
                 request = createTransmissionPostRequest(transmission);
                 httpClient.enhanceRequest(request);
-                response = httpClient.sendPostRequest(request);
+                response = httpClient.sendRequest(request);
```

Comment:
I checked HttpPost (HttpEntityEnclosingRequestBase) is not a type of HttpUriRequest, which is HttpRequest, and HttpMessage.  I don't see other changes related to it.. line 165 returns HttpPost? where is the magic?

Reply:
HttpPost extends HttpEntityEnclosingRequestBase extends HttpRequestBase implements HttpUriRequest 😄